### DISCUSSION TITLE
Dashboard run multi tasks

### DISCUSSION THREAD

**Discussion by MinhOmega** [ixahmedxi/orbitkit#93] [Q&A] [answered]
Hi authors,

I want to find out what technology you use to get the image I attached below

![CleanShot 2024-07-03 at 13 03 31](https://github.com/ixahmedxi/orbitkit/assets/49482201/14f87fc7-6965-43c9-bdfd-e87e0df3155b)

Thank in advance!

**Comment by imCorfitz** [comment, marked as answer]
This is the UI of the new 2.0 version of [Turbo](https://turbo.build/repo).
